installing/installing_aws/installing-restricted-networks-aws-installer-provisioned.adoc

```diff
@@ -0,0 +1,72 @@
+[id="installing-restricted-networks-aws-installer-provisioned"]
+= Installing a cluster on AWS in a restricted network
+include::modules/common-attributes.adoc[]
+:context: installing-restricted-networks-aws-installer-provisioned
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster on Amazon Web Services (AWS) in a restricted network by creating an internal mirror of the installation release content on an existing Amazon Virtual Private Cloud (VPC).
+
+[id="prerequisites_installing-restricted-networks-aws-installer-provisioned"]
+== Prerequisites
+
+* You xref:../../installing/install_config/installing-restricted-networks-preparations.adoc#installing-restricted-networks-preparations[created a mirror registry on your mirror host] and obtained the `imageContentSources` data for your version of {product-title}.
++
+[IMPORTANT]
+====
+Because the installation media is on the mirror host, you can use that computer to complete all installation steps.
+====
+* You created your own VPC in AWS hosting the mirror registry. When installing to a restricted network using installer-provisioned infrastructure, you cannot use the installer-provisioned VPC; your mirror registry must be hosted on a user-provided VPC configured for the cluster to use.
```

Comment:
This is not entirely accurate. The mirror registry could be hosted elsewhere. The point of needing to BYO VPC is that either (1) the mirror registry needs to be in the VPC or (2) the VPC needs to have firewall rules or peering to access the mirror registry hosted elsewhere. In either case, the VPC needs to exist before installation.